ArcPicker/ArcPicker.js

```diff
@@ -185,8 +193,10 @@ const ArcPickerBase = kind({
 	computed: {
 		arcSegments: (props, context) => {
 			const {accent: accentColor} = context || {};
-			const {children, endAngle, isFocused, onClick, radius, selectionType, skinVariants, startAngle, strokeWidth, value} = props;
-			const backgroundColor = props.backgroundColor || (skinVariants && skinVariants.night ? '#444444' : '#888888');
+			const {children, endAngle, isFocused, onClick, radius, selectionType, skin, skinVariants, startAngle, strokeWidth, value} = props;
+
+			const backgroundColorNight = skin === 'titanium' ? '#fff' : '#888888';
```

Comment:
can you use the 6 characters hex format for the color please?

Reply:
Solved